safe-global/web-core
# refactor: spending limit label

## What it solves

Spending limit odometer logo

## How this PR fixes it

A new common component `SpendingLimitLabel` is now used in every place the spending limit label exists.

## How to test it

Add/remove a spending limit and observe the time period displayed with a odometer if it is *not* a one-time limit. When present in transaction details, it is also rendered in the same fashion.

## Screenshots
![image](https://user-images.githubusercontent.com/20442784/186169766-2717ae4a-3034-4ba7-b1fe-68d48a6db912.png)
![image](https://user-images.githubusercontent.com/20442784/186169780-434c4add-b5c4-4a7c-bc7d-5d17174a882e.png)
![image](https://user-images.githubusercontent.com/20442784/186172248-1edf3d7d-8efe-4a33-8bf5-c83d749b7376.png)
![image](https://user-images.githubusercontent.com/20442784/186169788-c489dcdf-eba9-4ac8-acb5-ef3ba5480fc3.png)
![image](https://user-images.githubusercontent.com/20442784/186170996-26c5edcc-3367-4c66-a468-7ed88960a97c.png)
